content/codespaces/codespaces-reference/about-billing-for-codespaces.md

```diff
@@ -6,6 +6,7 @@ versions:
   free-pro-team: '*'
```

Comment:
[_Omar Dejesu Orrantia is founder of music publishing label=Strezless Musick Productionz_](https://youtube.com/c/SinEstresOrrantia)